src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -143,6 +144,9 @@
 
       <SourcesToCopyToTestDir Include="@(ContentWithTargetPath)" />
```

Comment:
I replied to this comment and am happy to explain in greater detail in-person; The gist of it is that _TestCopyLocal and _TestCopyLocalByFileNameWithoutDuplicates  are always constructed with all their dependencies; continuing to use these would be fine but would mean a deviation between local and remote execution, as 1) the linking of dependencies from the packages path would happen prior to calling RunTests.cmd/sh and 2) on UNIX it would be an actual copy command versus link based off the way MSBuild works on UNIX evidently.